tensorflow/lite/micro/cortex_m_corstone_300/avh.yml

```diff
@@ -0,0 +1,13 @@
+name: "TensorFlow Lite Micro Corstone-300 ArmClang"
```

Comment:
I feel this file might be better placed in .github/cortext_mcorstone_300 folder instead of tensorflow/lite/micro/cortex_m_corstone_300.  What is in tensorflow/lite/micro/cortex_m_corstone_300 are necessary initialization codes for building targets. This file is a configuration and is more about executing task in virtual machine remotely.

Reply:
I agree that the source folder is not ideal. I am a bit afraid that .github with its special purpose might cause issues someday, when Github accidently picks up the yml files and tries to interpret them. What do you think about: tensorflow/lite/micro/tools/arm-virtual-hardware/* as a place where we could place all related files (currently it will be a yml for each AVH workflow)?

Reply:
Sounds good to me in general. What do you think about: tensorflow/lite/micro/tools/github/arm_virtual_hardware/?  Prefer to use underscore _ instead of - to be consistent with the convention inside this folder.

Reply:
Done :)

Reply:
@MatthiasHertel80 I have approved the PR and noticed that it did not get check in b/c formatting issues. Can you please run tensorflow/lite/micro/tools/ci_build/test_code_style.sh --fix_formatting? That should get the format clear. You can also run tensorflow/lite/micro/tools/ci_build/test_code_style.sh afterwards to make sure there are not format issues.

Reply:
@MatthiasHertel80 It appears that your branch is out of date and that appears to be the reason that it cannot be merged (or so I guess since this is the only signal on the page).  I tried to update it, but for some reasons, your branch is not updatable by me.  Could you pleas sync to main and push again?